doc/source/dev/conduct/code_of_conduct.rst

```diff
@@ -0,0 +1,185 @@
+   contains the other messages with arguments that have already been made.
+
+7. Try to stay on topic, especially in discussions that are already fairly
+   large.
+
+8. Step down considerately. Members of every project come and go. When somebody
+   leaves or disengages from the project they should tell people they are
+   leaving and take the proper steps to ensure that others can pick up where
+   they left off. In doing so, they should remain respectful of those who
+   continue to participate in the project and should not misrepresent the
+   project's goals or achievements. Likewise, community members should respect
+   any individual's choice to leave the project.
+
+
+Diversity Statement
+-------------------
+
+The SciPy project welcomes and encourages participation by everyone. We are
+committed to being a community that everyone feels good about joining. Although
+we may not be able to satisfy everyone, we will always work to treat everyone
+well.
+
+No matter how you identify yourself or how others perceive you: we welcome you.
+Though no list can hope to be comprehensive, we explicitly honour diversity in:
+age, culture, ethnicity, genotype, gender identity or expression, language,
+national origin, neurotype, phenotype, political beliefs, profession, race,
+religion, sexual orientation, socioeconomic status, subculture and technical
+ability.
+
+Though we welcome people fluent in all languages, SciPy development is
+conducted in English.
+
+Standards for behaviour in the SciPy community are detailed in the Code of
+Conduct above. We expect participants in our community to meet these standards
+in all their interactions and to help others to do so as well.
+
+
+Reporting Guidelines
+--------------------
+
+While this code of conduct should be adhered to by participants, we recognize
+that sometimes people may have a bad day, or be unaware of some of the
+guidelines in this code of conduct. When that happens, you may reply to them
+and point out this code of conduct. Such messages may be in public or in
+private, whatever is most appropriate. However, regardless of whether the
+message is public or not, it should still adhere to the relevant parts of this
+code of conduct; in particular, it should not be abusive or disrespectful.
+
+If you believe someone is violating this code of conduct, you may reply to them
+and point out this code of conduct. Such messages may be in public or in
+private, whatever is most appropriate. Assume good faith; it is more likely
+that participants are unaware of their bad behaviour than that they
+intentionally try to degrade the quality of the discussion. Should there be
+difficulties in dealing with the situation, you may report your compliance
+issues in confidence to either:
+
+- The SciPy Code of Conduct Committee, at conduct@TODO (preferred)
+  Members of this committee are TODO
+
+Or alternatively to:
+
+- Chair of the SciPy Steering Committee: Ralf Gommers
```

Comment:
Perhaps add these two entries to CoC Committee

Reply:
These are kept separate on purpose in case someone does not want to contact the Code of Conduct Committee for some reason (e.g. one of the members of that committee is involved).

I could be on the CoC committee, but don't have to be. Leah shouldn't be on the CoC committee, she's not really part of the project - she's there as NumFOCUS director to have an outside contact in case someone feels they can't contact any of the SciPy committee or leadership.

Reply:
Thanks for clarifying.

Reply:
Replying for completeness: we discussed this in a call today, the outcome was yes we do need/want a CoC committee, ideally of 3-5 people.